packages/eslint-plugin-lit-a11y/README.md

```diff
@@ -2,7 +2,7 @@
 
 Accessibility linting plugin for lit-html.
 
-Most of the rules are ported from [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y).
+Most of the rules are ported from [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y), and made to work with [lit-html](https://lit-html.polymer-project.org/) templates.
```

Comment:
```suggestion
Most of the rules are ported from [eslint-plugin-jsx-a11y](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y), and made to work with HTML and [lit-html](https://lit-html.polymer-project.org/) templates.
```

Reply:
it doesnt work with plain HTML templates though :p

Reply:
it does in lit-html templates

the intention here was more to the "custom elements" part of HTML though, e.g. props -> attributes, HOCs -> slots

Reply:
maybe 

> and made to work with [lit-html](https://lit-html.polymer-project.org/) templates and custom elements.

then ?

I dont want to give people the idea that this eslint plugin works for plain html

Reply:
> made to work with custom elements and ...
?

Reply:
I'd put lit-html first because its the primary usecase

Reply:
updated the suggestion